pkg/controller/worker/machines.go

```diff
@@ -166,6 +165,7 @@ func (w *workerDelegate) generateMachineConfig() error {
 				},
 				"blockDevices":            blockDevices,
 				"instanceMetadataOptions": instanceMetadataOptions,
+				"srcAndDstChecksEnabled":  false,
```

Comment:
Should we maybe just embed that directly in the machineclass chart instead since it doesn't make sense for it to be configurable anyway ?